sklearn/inspection/_plot/tests/test_boundary_decision_display.py

```diff
@@ -577,18 +576,97 @@ def test_class_of_interest_multiclass(pyplot, response_method):
             class_of_interest=class_of_interest_idx,
         )
 
-    # TODO: remove this test when we handle multiclass with class_of_interest=None
-    # by showing the max of the decision function or the max of the predicted
-    # probabilities.
-    err_msg = "Multiclass classifiers are only supported"
-    with pytest.raises(ValueError, match=err_msg):
-        DecisionBoundaryDisplay.from_estimator(
-            estimator,
-            X,
-            response_method=response_method,
-            class_of_interest=None,
+
+@pytest.mark.parametrize("response_method", ["predict_proba", "decision_function"])
+def test_multiclass_plot_max_class(pyplot, response_method):
+    """Check plot correct when plotting max multiclass class."""
+    import matplotlib as mpl
+
+    # In matplotlib < v3.5, default value of `pcolormesh(shading)` is 'flat',
+    # which results in the last row and column of the color mapped values being
+    # dropped. This results in a 99x99 grid.
+    if parse_version(mpl.__version__) < parse_version("3.5"):
+        pytest.skip(
+            "`pcolormesh` in Matplotlib >= 3.5 gives smaller grid size."
```

Comment:
So min mpl version problem again.

Alternatively, I can add an if/else and for mpl < 3.5 and amend the test to something like:

```python
highest_class_mask = highest_class != idx
assert_array_equal(highest_class_mask[:-1,:-1].flatten(), quadmesh.get_array().mask)
```

and I can have a comment to delete this once we bump the min version of mpl (currently 3.3.4)